openslides_backend/action/actions/speaker/end_speech.py

```diff
@@ -26,12 +27,43 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
         for instance in action_data:
             speaker = self.datastore.fetch_model(
                 FullQualifiedId(self.model.collection, instance["id"]),
-                mapped_fields=["begin_time", "end_time"],
+                mapped_fields=["begin_time", "end_time", "meeting_id"],
                 lock_result=True,
             )
             if speaker.get("begin_time") is None or speaker.get("end_time") is not None:
                 raise ActionException(
                     f"Speaker {instance['id']} is not speaking at the moment."
                 )
             instance["end_time"] = round(time.time())
+
+            # reset projector_countdown
+            if speaker.get("meeting_id"):
```

Comment:
Also same as in #561: the speaker must have a meeting id.

Reply:
Removed this check.